extensions/amp-autocomplete/0.1/amp-autocomplete.js

```diff
@@ -422,6 +420,79 @@ export class AmpAutocomplete extends AMP.BaseElement {
     return this.truncateToMaxEntries_(filteredData);
   }
 
+  /**
+   * Returns true if the given input string is a token-prefix match on the
+   * given item string. Assumes toLocaleLowerCase() has been performed on both
+   * parameters.
+   * @param {string} item
+   * @param {string} input
+   * @return {boolean}
+   * @private
+   */
+  tokenPrefixMatch_(item, input) {
+    if (input === '') {
+      return true;
+    }
+
+    const itemTokens = this.tokenizeString_(item);
+    const inputTokens = this.tokenizeString_(input);
+
+    // Match each input token (except the last one) to an item token
+    const itemTokensMap = this.mapFromTokensArray_(itemTokens);
+    const lastInputToken = inputTokens[inputTokens.length - 1];
+    inputTokens.splice(inputTokens.length - 1, 1);
+    let match = true;
+    inputTokens.forEach(token => {
+      if (token === '') {
+        return;
+      }
+      if (!itemTokensMap.has(token)) {
+        return match = false;
```

Comment:
I don't want to exit the loop entirely, just move on from the current iteration. I think `return` achieves this but I will swap the `forEach/break` with a regular `for/continue` just in case.